video/csputils.h

```diff
@@ -67,6 +67,10 @@ enum mp_csp_prim {
     MP_CSP_PRIM_DISPLAY_P3,
     MP_CSP_PRIM_V_GAMUT,
     MP_CSP_PRIM_S_GAMUT,
+    MP_CSP_PRIM_EBU_3213,
+    MP_CSP_PRIM_FILM_C,
+    MP_CSP_PRIM_ACES_AP0,
+    MP_CSP_PRIM_ACES_AP1,
```

Comment:
Adding new enums require more changes if few places to handle them.

Reply:
Ah, you're right, looking at #11200. Thanks for the heads up